### PR TITLE
CTECH-3164: increase timeout on threading example

### PR DESCRIPTION
**Pull request by adamMQM**
# Pull Request Checklist

- [x] Read the [contributing guidelines](../blob/master/docs/CONTRIBUTING.md)
- [x] Tests pass

# Description of the PR

Increase the timeout on multi-threading example.

### CODE REVIEW

File: src/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,7 @@
-FROM python:3.7.0
+FROM python:3.11.3
 
 WORKDIR /usr/src/
 
-RUN apt-get update && apt-get -y install jq
-
```
Comment (pms1969) [inline src/Dockerfile:6]: I still think you want to keep this.  It might be up to date today, but it may not be tomorrow.

**Review by rizwansaeed — APPROVED** [review]

File: src/Dockerfile (hunk 1 above)
Comment (adamMQM) [reply src/Dockerfile:6]: This image is just for running the tests right? If so I don't believe jq is used at all

**Pull request merged**